scripts/create_gitlab_snapshot.py

```diff
@@ -120,6 +147,35 @@ def create_snapshot(volume: JSON):
                 WaiterConfig=dict(MaxAttempts=9999, Delay=15))
     log.info('Snapshot %r of volume %r is complete',
              snapshot_id, volume['VolumeId'])
+    return snapshot_id
+
+
+def create_volume(snapshot_id: str, size: int, volume: JSON):
+    new_volume = aws.ec2.create_volume(AvailabilityZone=volume['AvailabilityZone'],
+                                       Encrypted=volume['Encrypted'],
+                                       KmsKeyId=volume['KmsKeyId'],
+                                       Size=size,
+                                       SnapshotId=snapshot_id,
+                                       VolumeType=volume['VolumeType'],
+                                       TagSpecifications=[dict(ResourceType='volume', Tags=volume['Tags'])],
+                                       MultiAttachEnabled=volume['MultiAttachEnabled'])
+    log.info('New volume %r is being created using snapshot %r …',
+             new_volume['VolumeId'], snapshot_id)
+    waiter = aws.ec2.get_waiter('volume_available')
+    waiter.wait(VolumeIds=[new_volume['VolumeId']],
+                WaiterConfig=dict(MaxAttempts=9999, Delay=15))
+    log.info('New volume %r of size %s GiB is ready to attach',
+             new_volume['VolumeId'], new_volume['Size'])
+    tags = volume['Tags']
+    assert isinstance(tags, list), tags
+    _untag_volume(volume['VolumeId'], tags)
+
+
+def _untag_volume(old_volume_id: str, tags: list[str]):
+    log.info('Removing tags from volume %r …', old_volume_id)
+    aws.ec2.delete_tags(Resources=[old_volume_id], Tags=tags)
+    log.info('Tags removed.\nSave this command, do not run it prematurely:\n'
+             '# aws ec2 delete-volume --volume-id %r', old_volume_id)
```

Comment:
Generally, we don't format log messages, and let the console wrap them. 

Also, this log statement should be in the method above and then I doubt that we even need this method as there is only one call site.